packages/popover/src/Popover.tsx

```diff
@@ -20,7 +20,6 @@ import {
 const rootPopoverStyle = css`
   position: absolute;
   transition: transform 150ms ease-in-out, opacity 150ms ease-in-out;
-  transform: translate3d(0, 0, 0) scale(1);
```

Comment:
Curious why this was causing the clipping behavior

Reply:
I believe because it was overriding the scale amount of 0.8 defined in positionUtils.ts